### ISSUE TITLE
Audiences attributes only work with string values

### ISSUE THREAD

**Issue by FelipeLahti** [optimizely/javascript-sdk#146]
`optimizely.isFeatureFlag('flag-key', 'user-id', { isAdmin: true})` only get segmented in the isAdmin audience if the attribute has the type string `"true"` and doesn't actually work with booleans.

i.e `optimizely.isFeatureFlag('flag-key', 'user-id', { isAdmin: true})` returns false
i.e `optimizely.isFeatureFlag('flag-key', 'user-id', { isAdmin: "true"})` returns true

**Comment by mikeng13** [comment]
@FelipeLahti thanks for submitting your feedback. The SDK currently only supports string types when matching audience conditions. We are actively working on expanding this to support other types such as booleans. Please stay tuned!

**Comment by FelipeLahti** [comment]
Thanks @mikeng13 
As an workaround, I encapsulated the call to `isFeatureFlag` and I'm converting each field to string so another developers on my team doesn't have to go through the same issue I had.
Something like:
```javascript
// https://github.com/optimizely/javascript-sdk/issues/132
function getUserId(userId) {
  return userId === undefined ? 'userId' : userId;
}

// https://github.com/optimizely/javascript-sdk/issues/146
function getUserAttributes(userAttributes) {
  if (userAttributes) {
    const userAtributtesStringify = {};
    Object.keys(userAttributes).forEach((key) => {
      if (userAttributes[key] !== undefined) {
        userAtributtesStringify[key] = userAttributes[key].toString();
      }
    });
    return userAtributtesStringify;
  }
  return userAttributes;
}

function isFeatureEnabled(flagKey, userId, audiences) {
  return optimizely.isFeatureEnabled(flagKey, getUserId(userId), getUserAttributes(audiences));
}
```

**Comment by mikeng13** [comment]
We have released a 3.0.0-alpha that has support for non string attributes. You will however, have to take to your csm about enabling audience targeting with non string attributes in the UI.
